src/pybind/mgr/dashboard/tox.ini

```diff
@@ -1,40 +1,27 @@
 [tox]
-envlist = cov-init,py27,py3,cov-report,lint
+envlist = {py27,py3}-cov,{py27,py3}-lint
 skipsdist = true
 
 [testenv]
-deps=-r{toxinidir}/requirements.txt
+basepython =
+    py27: python2.7
+    py3: python3
```

Comment:
For some unknown reason, the virtual environments are being created outside of `.tox` directory, please add this additional line to fix it:
`envdir = {toxinidir}/.tox/{envname}`

Reply:
@rjfd it's on purpose. i wanted to avoid creating the venvs in the source directory. see https://github.com/ceph/ceph/pull/21293/files#diff-cc2ee9d8e56f3a2cd98b8148935d3829R28.  why do you think we should put them into `.tox`?

Reply:
I see. I usually run `run-tox.sh` manually in the source directory and I was getting unstaged changes in my repo because of those directories. I was suggesting to move them into `.tox` directory because .tox is in the .gitignore.

Instead of creating them inside .tox, we can add the virtualenv directories to the .gitignore?